update FlexQuery field names and ordering

alensiljak opened the pull request.

redstreet commented:
Hmm, I use this importer all the time including just a few minutes ago, and it works fine as it is. I haven't looked at depth into your PR, but I'd suspect this would break it for me. Is this because of how you have your config setup at your IBKR account?

alensiljak commented:
Hm, interesting. I'm not sure I understand your question. This is the order of fields I get when creating a new Flex query. I just changed the instructions to list the fields in the order they appear in the list, so it is easier to select, rather than jumping around. I don't think I changed any code in this one.

alensiljak commented:
Also, these are the field names in the UI. I wonder if they are the same in the XML.

redstreet commented:
Got it, nvm my earlier comment, my bad. Merging!
